internal/config/wrapper.go

```diff
@@ -230,6 +230,7 @@ func (w *Wrapper) InvalidateFolder(id string, err string) {
 	for i := range w.cfg.Folders {
 		if w.cfg.Folders[i].ID == id {
 			w.cfg.Folders[i].Invalid = err
+			events.Default.Log(events.ConfigSaved, w.cfg)
```

Comment:
Yeah, this is more telling the GUI to refresh the config which now has a 'Invalid' set.
Should we add a separate event for refresh config or perhaps FolderError or something?